pkg/pagerduty/pagerduty_test.go

```diff
@@ -0,0 +1,83 @@
+package pagerduty
+
+import (
+	"time"
+
+	pagerduty "github.com/PagerDuty/go-pagerduty"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Pls don't use `v2` yet , we may need to change all tests then

Reply:
Hi Sam, fixing this is causing goroutine error
```
/var/folders/6k/x9lbvrdd25g6fb44_rzzv79c0000gn/T/go-build3713586784/b001/pagerduty.test flag redefined: ginkgo.seed
panic: /var/folders/6k/x9lbvrdd25g6fb44_rzzv79c0000gn/T/go-build3713586784/b001/pagerduty.test flag redefined: ginkgo.seed

goroutine 1 [running]:
flag.(*FlagSet).Var(0x140001ce150, {0x10502f5b0, 0x1052840e0}, {0x140001a0f10, 0xb}, {0x104ed41f6, 0x2a})
        /opt/homebrew/Cellar/go/1.21.7/libexec/src/flag/flag.go:1028 +0x308
flag.(*FlagSet).Int64Var(...)
        /opt/homebrew/Cellar/go/1.21.7/libexec/src/flag/flag.go:801
github.com/onsi/ginkgo/v2/types.bindFlagSet({{0x1400028a000, 0x29, 0x40}, {0x104fb9bc0, 0x1400027a780}, {0x105266e80, 0xd, 0xd}, {{0x0, 0x0}, ...}, ...}, ...)
        /Users/dmaharan/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.19.0/types/flags.go:162 +0x574
github.com/onsi/ginkgo/v2/types.NewAttachedGinkgoFlagSet(...)
        /Users/dmaharan/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.19.0/types/flags.go:114
github.com/onsi/ginkgo/v2/types.BuildTestSuiteFlagSet(0x1052840e0, 0x105283c00)
        /Users/dmaharan/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.19.0/types/config.go:373 +0x6b0
github.com/onsi/ginkgo/v2.init.0()
        /Users/dmaharan/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.19.0/core_dsl.go:47 +0x4c
exit status 2
FAIL    github.com/openshift/backplane-cli/pkg/pagerduty        0.474s
```